### DISCUSSION TITLE
Who's using Prisma with Supabase? Do you know how to run `prisma migrate reset` without losing the Supabase configuration?

### DISCUSSION THREAD

**Discussion by lumenwrites** [prisma/prisma#16566] [Q&A]
Hey guys! I'm using prisma with Supabase. Most things work great, the problem is that when I'm using Prisma in development, it requires me to drop the database from time to time.

Every time I drop the database, the Supabase project loses its config - RLS policies, trigger functions, hooks, some permissions. It takes me like 40 minutes to manually resetup everything every time.

Do you know if there's a way around that? If you have some experience with Prisma and Supabase, can you share some advice?

**Comment by pantharshit00** [comment]
Hello @lumenwrites,

I am not very familiar with Supabase but if they store the configuration in the database itself, you maybe take out a [sql dump](https://www.prisma.io/dataguide/postgresql/inserting-and-modifying-data/importing-and-exporting-data-in-postgresql) of the database and store all this configuration in a migration which you can create using `--create-only` flag and it will reapply these configuration whenever you will recreate the database.

**Comment by janpio** [comment]
Can you open a feature request issue for this @lumenwrites? We should define exactly what needs to stay and then find a way to make that happen. Your help would be greatly appreciated.

**Reply by lumenwrites** [reply]
[Done](https://github.com/prisma/prisma/issues/16588)!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
